vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/PasswordField.java

```diff
@@ -56,13 +42,8 @@
 @JsModule("@vaadin/polymer-legacy-adapter/style-modules.js")
 @NpmPackage(value = "@vaadin/password-field", version = "24.0.0-alpha6")
 @JsModule("@vaadin/password-field/src/vaadin-password-field.js")
-public class PasswordField
-        extends GeneratedVaadinTextField<PasswordField, String> implements
-        CompositionNotifier, Focusable<PasswordField>, HasAllowedCharPattern,
-        HasAutocapitalize, HasAutocomplete, HasAutocorrect, HasClearButton,
-        HasClientValidation, HasHelper, HasLabel, HasPrefixAndSuffix, HasSize,
-        HasStyle, HasThemeVariant<TextFieldVariant>, HasTooltip, HasValidation,
-        HasValidator<String>, HasValueChangeMode, InputNotifier, KeyNotifier {
+public class PasswordField extends InternalFieldBase<PasswordField, String>
```

Comment:
Looks like the value change mode logic can be removed here as well?

Reply:
Thanks, I missed to remove it. Done.